### PR TITLE
Docs: Ukrainian — Add Sessions&Cookies page

### PR DESCRIPTION
**Pull request by 101000june**
Fully added Ukrainian language support for Sessions&Cookies page, for www.yiiframework.com/doc/guide
| Q                    | A
| --------------- | ---
| Is bugfix?        | ❌
| New feature?  | ✔️
| Breaks BC?      | ❌
| Fixed issues     | no Ukrainian "Sessions" page

### CODE REVIEW

File: docs/guide-uk/runtime-sessions-cookies.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
+Сесії і кукі
```
Comment (samdark) [inline docs/guide-uk/runtime-sessions-cookies.md:1]:
```suggestion
Сесії та кукі
```

File: docs/guide-uk/runtime-sessions-cookies.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+====================
+
+Сесії та кукі дозволяють зберігати користувацькі дані між запитами. При використанні чистого PHP можна отримати доступ до цих даних через глобальні змінні `$_SESSION` та `$_COOKIE`, відповідно. Yii інкапсулює сесії та кукі в об'єкти, що дає можливість звертатися до них в об'єктно-орієнтованому стилі та забезпечує додаткову зручність в роботі.
```
Comment (samdark) [inline docs/guide-uk/runtime-sessions-cookies.md:4]:
```suggestion
Сесії та кукі дозволяють зберігати користувацькі дані між запитами. При використанні чистого PHP можна отримати доступ до цих даних через глобальні змінні `$_SESSION` та `$_COOKIE`, відповідно. Yii інкапсулює сесії та кукі в об'єкти, що дає можливість звертатися до них в об'єктноорієнтованому стилі та забезпечує додаткову зручність в роботі.
```

File: docs/guide-uk/runtime-sessions-cookies.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+
+## Сесії <span id="sessions"></span>
+
+За аналогією з [запитами](runtime-requests.md) та [відповідями](runtime-responses.md), до сесій можна отримати доступ через `session` [компонент додатку](structure-application-components.md), який за замовчуванням є екземпляром [[yii\web\Session]].
```
Comment (samdark) [inline docs/guide-uk/runtime-sessions-cookies.md:9]:
```suggestion
За аналогією з [запитами](runtime-requests.md) та [відповідями](runtime-responses.md), до сесій можна отримати доступ через `session` [компонент додатка](structure-application-components.md), який за замовчуванням є екземпляром [[yii\web\Session]].
```

**Review by samdark — APPROVED** [review]
:+1: Please re-check suggestions.

**Review by Arhell — APPROVED** [review]

**Comment by 101000june** [comment]
applied suggestions

**Comment by samdark** [comment]
Thank you!

**Comment by 101000june** [comment]
> Thank you!

Please tell me, will it be applied soon? I don't see any changes now, still English at https://www.yiiframework.com/doc/guide/2.0/uk/runtime-sessions-cookies

**Comment by samdark** [comment]
It is rebuilt from time to time. Don't worry. Changes will be there.

**Pull request merged**